packages/cozy-harvest-lib/test/components/KonnectorModal.spec.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-env jest */
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { KonnectorModal } from 'components/KonnectorModal'
+import TriggerManager from 'components/TriggerManager'
+
+const t = jest.fn().mockImplementation(key => key)
+
+const findAccountMock = jest.fn().mockImplementation(() => ({
+  id: 123,
+  doctype: 'io.cozy.accounts'
+}))
```

Comment:
No need for a mock function here either.

Reply:
We need to correctly mock the fetching account from the trigger provided

Reply:
As findAccount is asynchronous, I suggest: 
```
const findAcocuntMock = jest.fn().mockResolvedValue({
  id: 123,
  doctype: 'io.cozy.accounts'
})
```

Reply:
I dont undertand why we use jest.fn here ? 

```
const findAccountMock = () => Promise.resolve({ id: 123, doctype: 'io.cozy.accounts' })
```

It is not really a problem, but I see no point in using jest.fn when we do not need it.
```

Reply:
Since we don't await fetchAccount in the componentDidMount, it doesn't work